### PR TITLE
[SELS-TASK] - [FE] - Mockup admin login and register page 

### PR DESCRIPTION
**Pull request by Mjdelima20**
**Asana** 
https://app.asana.com/0/1205809759545605/1205809759545626/f

**[Commands]**

- `npm start`

**[Pre-conditions]**

1.  After running the command , you will redirect to front page
2.  Click  Admin
3.  User will Redirect  to Admin Sign in Page
4. Click Register as Admin
5. User will redirect to Admin Sign up Page

**Screenshots**
<Details> 

image

image

image

</Details>

### CODE REVIEW

File: frontend/src/Pages/auth/AdminLogin.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      </div>
+  </div>
+</section> 
+    </>
+    )
+}
+
+
+
+export default AdminLogin;
```
Comment (RDLegs) [inline frontend/src/Pages/auth/AdminLogin.tsx:45]: Add new line to EOF

File: frontend/src/Pages/auth/AdminSignUp.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+              <div className="bg-gray-800 border-gray-400 border-[1px] rounded shadow-lg p-9 px-4 md:p-8 mb-6">
+                <div className=" grid gap-4 gap-y-2 text-sm grid-cols-1 lg:grid-cols-3">
+                  <div className="text-gray-600">
+                    <p className="font-medium text-lg text-gray-300">Admin Details</p>
+                    <p className="text-gray-400">Please fill out all the fields.</p>
+                  </div>
+
+                  <div className="md:col-span-2 text-gray-300" >
+                    <div className="grid gap-4 gap-y-3 text-sm grid-cols-2 md:grid-cols-4">
+                      <div className="md:col-span-2">
```
Comment (RDLegs) [inline frontend/src/Pages/auth/AdminSignUp.tsx:25]: For all your inputs here, please create a `components` folder and extract this into a component.

Comment (Mjdelima20) [reply frontend/src/Pages/auth/AdminSignUp.tsx:25]: @RDLegs 
Good morning sir, I just want to clarify , if is it for adminSignUp page only ?  or should I also implement it to student login , studentSignup and admin login page?

Comment (RDLegs) [reply frontend/src/Pages/auth/AdminSignUp.tsx:25]: If it has a similar structure in those other pages, then yes implement it for those pages as well

**Review by RDLegs — APPROVED** [review]
LGTM!

**Pull request merged**